### DISCUSSION TITLE
python time.sleep function not working properley when deployed

### DISCUSSION THREAD

**Discussion by AvailibleName** [vercel/community#1187] [Help]
### Summary

My python web app uses a time.sleep(x) statement and works locally but when I deploy it on vercel, it sleeps the thread indefinitley, rather than for the specified time. I tried reducing the time to 1 second so that it isn't possible that something weird could be happening with timeouts and that didn't change anything. I also couldnt find any documentation regarding it.

**Comment by amyegan** [comment]
Hi @AvailibleName! Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use to get a better look at the problem?
